js/src/wp-seo-network-admin.js

```diff
@@ -0,0 +1,96 @@
+/* global wpseoNetworkAdminGlobalL10n, ajaxurl */
+
+import a11ySpeak from "a11y-speak";
+
+( function( $ ) {
+	"use strict";
+
+	/**
+	 * Displays given settings errors.
+	 *
+	 * @param {Object} settingsErrors The list of settings error objects.
+	 *
+	 * @returns {void}
+	 */
+	function displaySettingsErrors( settingsErrors ) {
+		var $heading = $( ".wrap > h1" );
+		var notices;
+		var prefix;
+
+		if ( ! settingsErrors.length ) {
+			return;
+		}
+
+		notices = settingsErrors.map( function( settingsError ) {
+			return "<div class='" + settingsError.type + " notice'><p>" + settingsError.message + "</p></div>";
+		} );
+
+		$heading.after( notices.join( "" ) );
+
+		prefix = wpseoNetworkAdminGlobalL10n.error_prefix;
+		if ( settingsErrors[ 0 ].type === "updated" ) {
+			prefix = wpseoNetworkAdminGlobalL10n.success_prefix;
+		}
+
+		a11ySpeak( prefix.replace( "%s", settingsErrors[ 0 ].message ), "assertive" );
+	}
+
+	/**
+	 * Handles a form submission with AJAX.
+	 *
+	 * @param {Event} event The submission event.
+	 *
+	 * @returns {void}
+	 */
+	function handleAJAXSubmission( event ) {
+		var $form    = $( this );
+		var $submit  = $form.find( "[type='submit']:focus" );
```

Comment:
Why getting the button (and thus its value to be use as action) only when focused? Browsers have slightly different behaviors when it comes to submitting forms using the keyboard. For example:
- go in the Restore Site tab
- with Chrome, I'm able to tab to the select element and focus it, press Enter and the form submits
- at that point, the button is not focused 
- the actual action is `yoast_handle_network_options` while it's supposed to be `yoast_restore_site`
- with Firefox:
- if I click the button using the mouse, the action is `yoast_handle_network_options`
- if I press the button using the keyboard, the action is `yoast_restore_site`

Reply:
It's necessary to find out through which button the form was submitted as that button determines the `action` to pass to the AJAX request (which on the server is then used as the AJAX action via `wp_ajax_{$action}`.

I adjusted the logic accordingly so that submitting without focusing a button still works:
* First, check if a button is focused. If so, use that.
* If not, find the current tab, and if there's a button inside that, use that.
* Otherwise fall back to the default action, which is for updating the options.